guacamole-common/src/main/java/org/apache/guacamole/protocol/ConfiguredGuacamoleSocket.java

```diff
@@ -184,6 +199,20 @@ public ConfiguredGuacamoleSocket(GuacamoleSocket socket,
                     "image",
                     info.getImageMimetypes().toArray(new String[0])
                 ));
+        
+        // Protocol version 1.1.0 and higher options
+        
+        if (protocol.atLeast(GuacamoleProtocolVersion.VERSION_1_1_0)) {
+            // Send client timezone, if available
+            String timezone = info.getTimezone();
+            if (timezone != null && !timezone.isEmpty()) {
```

Comment:
Any reason why both `null` and `""` are used rather than just `null`?

Reply:
Well, currently in the `GuacamoleClientInformation` class it is pre-initialized to an empty string.  I don't know if there was a particular reason I did this - I think I was trying to follow the pattern of the other config information, like the mimetypes, which are initialized to an empty list.  So, if anything, shouldn't it be only empty and skip the `null` check?

Reply:
Reduced it to just the `null` check/value.